x/dispensation/types/msgs.go

```diff
@@ -28,6 +28,7 @@ func (m MsgDistribution) Route() string {
 	return RouterKey
 }
 
+//TODO Replace with constant defined in keys.go with value CreateDispensation
```

Comment:
Should we address it in this PR?

Reply:
No , this is more of a cosmetic change , I put that there to keep track . Just trying to keep changes minimal right now , and add everything in bulk witht he .42 upgrade.